### PR TITLE
fix(clevertap): region undefined issue

### PR DESCRIPTION
**Pull request by mihir-4116**
## PR Description

Please include a summary of the change along with the relevant motivation and context.

## Linear task (optional)

Linear task link

## Cross Browser Tests

Please confirm you have tested for the following browsers:

- [ ] Chrome
- [ ] Firefox
- [ ] IE11

## Sanity Suite

- [ ] All sanity suite test cases pass locally

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

### CODE REVIEW

File: packages/analytics-js-integrations/src/integrations/Clevertap/browser.js [hunk 1]
```diff
@@ -67,6 +67,7 @@ class Clevertap {
       onUserLogin: [],
       notifications: [],
       privacy: [],
+      region: [],
```
Comment (saikumarrs) [inline packages/analytics-js-integrations/src/integrations/Clevertap/browser.js:70]: Incorrect.
As per their documentation, it's not an array and we have to define a default value.
https://developer.clevertap.com/docs/web-quickstart-guide#add-region-code

**Review by saikumarrs — CHANGES_REQUESTED** [review]

File: packages/analytics-js-integrations/src/integrations/Clevertap/browser.js (hunk 1 above)
Comment (mihir-4116) [reply packages/analytics-js-integrations/src/integrations/Clevertap/browser.js]: addressed, kept default region value as none

Comment (saikumarrs) [reply packages/analytics-js-integrations/src/integrations/Clevertap/browser.js]: We should not define any default value if the region is not explicitly provided by the user as per their docs. Sorry, my bad.

File: packages/analytics-js-integrations/src/integrations/Clevertap/browser.js [hunk 2]
```diff
@@ -67,6 +67,7 @@ class Clevertap {
       onUserLogin: [],
       notifications: [],
       privacy: [],
+      region: 'none',
```
Comment (saikumarrs) [inline packages/analytics-js-integrations/src/integrations/Clevertap/browser.js:70]: No default value should be set.

Comment (mihir-4116) [reply packages/analytics-js-integrations/src/integrations/Clevertap/browser.js]: addressed

**Review by saikumarrs — APPROVED** [review]

**Review by yashasvibajpai — APPROVED** [review]

File: packages/analytics-js-integrations/__tests__/integrations/Clevertap/browser.test.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import Clevertap from '../../../src/integrations/Clevertap/browser';
```
Comment (koladilip) [inline packages/analytics-js-integrations/__tests__/integrations/Clevertap/browser.test.js:1]: I saw region in other destinations also, can this be done as generic test?

Comment (mihir-4116) [reply packages/analytics-js-integrations/__tests__/integrations/Clevertap/browser.test.js:1]: i have added it in our notion doc, let's not do common test in this pr as it differs the purpose of this pr. tests are covered for the changes i have made. will add region common test in other pr which i raised

**Review by koladilip — APPROVED** [review]

**Pull request merged**